.github/workflows/android.yml

```diff
@@ -128,37 +143,24 @@ jobs:
         env:
           GST_VERSION: 1.18.5
         run: |
-            wget --quiet https://gstreamer.freedesktop.org/data/pkg/android/${GST_VERSION}/gstreamer-1.0-android-universal-${GST_VERSION}.tar.xz
+            wget https://gstreamer.freedesktop.org/data/pkg/android/${GST_VERSION}/gstreamer-1.0-android-universal-${GST_VERSION}.tar.xz
```

Comment:
How come removal of quiet. Without this it makes lookout the the raw logs really tough with a massive amount of wget logs.

Reply:
I had an issue at one point where it wasn't downloading correctly and I couldn't tell but I can put it back.

Reply:
Would be better without it. It ends up dumping page after page of downloading progress.